packages/backend/src/index.ts

```diff
@@ -47,3 +50,29 @@ logClients();
 telemetryFeed.on('connection', (socket: WebSocket) => {
   aggregator.addFeed(new Feed(socket));
 });
+
+http.createServer((request, response) => {
+  const incoming_url = request.url || "";
+  const parsed_url = url.parse(incoming_url, true);
+  const path = parsed_url.path || "";
+  if (path.startsWith("/network_state/")) {
+    const [chain_id, str_node_id] = path.split('/').slice(2);
+    console.log(chain_id, str_node_id);
+    const node_id = Number(str_node_id);
+    const chain = aggregator.getChain(chain_id as Types.ChainLabel);
+    const nodeList = Array.from(chain.nodeList());
```

Comment:
This, for some reason, is empty even after the node has connected to telemetry. Any ideas?